replica-healthcheck 0.3.9 docker won't start

**Describe the bug**

Fails with the following error message, appears to be missing contracts package

```
replica-healthcheck_1   | node:internal/modules/cjs/loader:936
replica-healthcheck_1   |   throw err;
replica-healthcheck_1   |   ^
replica-healthcheck_1   |
replica-healthcheck_1   | Error: Cannot find module '@eth-optimism/contracts'
replica-healthcheck_1   | Require stack:
replica-healthcheck_1   | - /opt/optimism/packages/sdk/dist/utils/contracts.js
replica-healthcheck_1   | - /opt/optimism/packages/sdk/dist/utils/index.js
replica-healthcheck_1   | - /opt/optimism/packages/sdk/dist/index.js
replica-healthcheck_1   | - /opt/optimism/packages/replica-healthcheck/dist/healthcheck-server.js
replica-healthcheck_1   | - /opt/optimism/packages/replica-healthcheck/dist/index.js
replica-healthcheck_1   | - /opt/optimism/packages/replica-healthcheck/dist/exec/run-healthcheck-server.js
replica-healthcheck_1   |     at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
replica-healthcheck_1   |     at Function.Module._load (node:internal/modules/cjs/loader:778:27)
replica-healthcheck_1   |     at Module.require (node:internal/modules/cjs/loader:1005:19)
replica-healthcheck_1   |     at require (node:internal/modules/cjs/helpers:102:18)
replica-healthcheck_1   |     at Object.<anonymous> (/opt/optimism/packages/sdk/dist/utils/contracts.js:4:21)
replica-healthcheck_1   |     at Module._compile (node:internal/modules/cjs/loader:1103:14)
replica-healthcheck_1   |     at Object.Module._extensions..js (node:internal/modules/cjs/loader:1155:10)
replica-healthcheck_1   |     at Module.load (node:internal/modules/cjs/loader:981:32)
replica-healthcheck_1   |     at Function.Module._load (node:internal/modules/cjs/loader:822:12)
replica-healthcheck_1   |     at Module.require (node:internal/modules/cjs/loader:1005:19) {
replica-healthcheck_1   |   code: 'MODULE_NOT_FOUND',
replica-healthcheck_1   |   requireStack: [
replica-healthcheck_1   |     '/opt/optimism/packages/sdk/dist/utils/contracts.js',
replica-healthcheck_1   |     '/opt/optimism/packages/sdk/dist/utils/index.js',
replica-healthcheck_1   |     '/opt/optimism/packages/sdk/dist/index.js',
replica-healthcheck_1   |     '/opt/optimism/packages/replica-healthcheck/dist/healthcheck-server.js',
replica-healthcheck_1   |     '/opt/optimism/packages/replica-healthcheck/dist/index.js',
replica-healthcheck_1   |     '/opt/optimism/packages/replica-healthcheck/dist/exec/run-healthcheck-server.js'
replica-healthcheck_1   |   ]
replica-healthcheck_1   | }
```

--- a/.changeset/early-pans-provide.md
+++ b/.changeset/early-pans-provide.md
@@ -1,0 +1,5 @@
+---
+'@eth-optimism/replica-healthcheck': patch
+---
+
+Fix bug in replica healthcheck dockerfile

--- a/.github/workflows/publish-canary.yml
+++ b/.github/workflows/publish-canary.yml
@@ -345,7 +345,7 @@ jobs:
           build-args: BUILDER_TAG=${{ needs.builder.outputs.canary-docker-tag }}
 
   replica-healthcheck:
-    name: Publish Data Transport Layer Version ${{ needs.builder.outputs.canary-docker-tag }}
+    name: Publish Replica Healthcheck Version ${{ needs.builder.outputs.canary-docker-tag }}
     needs: builder
     if: needs.builder.outputs.replica-healthcheck != ''
     runs-on: ubuntu-latest

--- a/ops/docker/Dockerfile.replica-healthcheck
+++ b/ops/docker/Dockerfile.replica-healthcheck
@@ -19,6 +19,11 @@ COPY --from=builder /optimism/packages/core-utils/dist ./packages/core-utils/dis
 COPY --from=builder /optimism/packages/common-ts/package.json ./packages/common-ts/package.json
 COPY --from=builder /optimism/packages/common-ts/dist ./packages/common-ts/dist
 
+COPY --from=builder /optimism/packages/contracts/package.json ./packages/contracts/package.json
+COPY --from=builder /optimism/packages/contracts/deployments ./packages/contracts/deployments
+COPY --from=builder /optimism/packages/contracts/dist ./packages/contracts/dist
+COPY --from=builder /optimism/packages/contracts/artifacts ./packages/contracts/artifacts
+
 # copy the service
 WORKDIR /opt/optimism/packages/replica-healthcheck
 COPY --from=builder /optimism/packages/replica-healthcheck/dist ./dist

